### ISSUE TITLE
Background Actualization of IDs only when App is started

### ISSUE THREAD

**Issue by tibor** [corona-warn-app/cwa-app-ios#1542]
## Avoid duplicates

* [x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is not already reported in another issue

## Describe the bug

With version 1.6+ the Corona warn app fails to update contact IDs in the background again, meaning the user has to open the app every day to keep the app working.

## Expected behaviour

The app should update IDs automatically in the background. 

## Steps to reproduce the issue

When opening the app, the app shows a last update timestamp larger than 24 hours ago. After some seconds the app updates the IDs then.

![image](https://user-images.githubusercontent.com/3136416/99570791-c2065d00-29d2-11eb-93e3-337f2460ea94.png)
![image](https://user-images.githubusercontent.com/3136416/99570813-c9c60180-29d2-11eb-90c0-256807f2d2fd.png)

## Technical details

- iOS Version: iOS 14.2
- App Version: 1.6.1
- Device: iPhone 11 Pro

## Additional context

The problem occurred in previous versions of the app but was fixed later. #740

**Comment by tibor** [comment]
Update: Seems like the app tries to update in the background, but the „New Files“ are empty.
![image](https://user-images.githubusercontent.com/3136416/99571270-583a8300-29d3-11eb-87ea-138761e29037.png)
![image](https://user-images.githubusercontent.com/3136416/99571285-5c66a080-29d3-11eb-9ea2-0f6bd812f922.png)
![image](https://user-images.githubusercontent.com/3136416/99571294-5e306400-29d3-11eb-8f09-97929d95d0fa.png)

**Comment by Ein-Tim** [comment]
My experience:

The background checks got more unreliable with 1.6.0/1.6.1, but it seems like the App sometimes still succeed to perform checks, but the real problem is: 
Dead Man messages are not send anymore, which is really bad. 

To be honest, IMHO this overall situation requires a hotfix.

**Comment by ndegendogo** [comment]
Same here. I reported it [here](https://github.com/corona-warn-app/cwa-app-ios/issues/916#issuecomment-729933255) in more detail.

A successfull check, followed by an aborted check (empty list) 4 hours later, **then two full days nothing**.
Then it started to check again, but no download of fresh files. Only after 4 more hours I got the successfull check of fresh files.

**Comment by dsarkar** [comment]
Dear @tibor and community,

Thanks for contributing. As pointed out by @ndegendogo and @Ein-Tim this seems a duplicate of https://github.com/corona-warn-app/cwa-app-ios/issues/916.  I suggest you continue this conversation there.

Thanks. Best wishes,
DS

---
Corona-Warn-App Open Source Team

**Comment by heinezen** [comment]
Duplicate of #916
